docs/sources/flow/reference/cli/tools.md

```diff
@@ -0,0 +1,80 @@
+---
+title: grafana-agent tools
+weight: 100
+---
+
+# `grafana-agent tools` command
+
+The `grafana-agent tools` command contains command line tooling grouped by Flow
+component.
+
+> **NOTE**: utilities in this command have no backwards compatibility
+> guarantees and may change or be removed between releases.
+
+## Subcommands
+
+### prometheus.remote_write sample-stats
+
+Usage: `grafana-agent tools prometheus.remote_write sample-stats [FLAG ...] WAL_DIRECTORY`
+
+The `sample-stats` command reads the Write-Ahead Log (WAL) specified by
+`WAL_DIRECTORY` and collects information on metric samples within it.
+
+For each metric discovered, `sample-stats` emits:
+
+* The timestamp of the oldest sample received for that metric.
+* The timestamp of the newest sample received for that metric.
+* The total number of samples discovered for that metric.
+
+By default, `sample-stats` will return information for every metric in the WAL.
+The `--selector` flag can optionally be passed to filter the reported metrics
+to a smaller set.
+
+The following flags are supported:
```

Comment:
Is this part of the template style? Since it's one flag, it should be `The following flag is supported:` but I can see the argument for a template style usecase here

Reply:
I don't think we necessarily have a defined style yet, so I'll change it to make it clearer that it's just a single flag 👍